src/perf/power.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: Preserving your battery
+description: Learn more about using limiting batter usage in your Flutter app.
+sitemap: false
```

Comment:
"batter usage" ? LOL

But, maybe we should just delete these TBD pages?

Reply:
I've been wanting to get into baking!

Yeah they've been sitting in this state for a while. I'll delete them and make an issue tracking interest for those pages.